contrib/core-contract-tests/Clarinet.toml

```diff
@@ -8,3 +8,10 @@ costs_version = 1
 [contracts.bns]
 path = "../../stackslib/src/chainstate/stacks/boot/bns.clar"
 depends_on = []
+epoch = 2.4
```

Comment:
This should probably be 2.0 right? (which is the default)

Reply:
no, because then we have two blocks on launch and all tests needs to be changed.
Using 2.4 simplifies the block counting.

We test in clarinet with 2.4, in rust we can do tests with different epochs as we do already.

Reply:
Ok, too bad.
I think we discussed the hard coded block heights in tests, but maybe we didn't fix it

Feel free to resolve then if you are ok we that

Reply:
We have the fix, but I think it should be in a different PR so that this one is small.